docs/release-notes/6.x.x.md

```diff
@@ -25,6 +25,8 @@ For additional information, please see [versioning](../versioning/semantic-versi
 - Python requirements:
   - AVD now requires the `cryptography` Python library to be version 43.0.0 or higher.<br>
     The latest version can be installed with `pip install "cryptography>=43.0.0" --upgrade`.
+- Ansible collection requirements:
+  - `arista.avd` Ansible collection does **not** depend anymore on `arista.cvp` and `ansible.utils` collections.
```

Comment:
will split ansible.utils in a different PR with porting guide entry making it clear whats going on